distribution/changelog.txt

```diff
@@ -1,5 +1,6 @@
 0.0.5 (in development)
 ------------------------------------------------------------------------
+- Feature: New cheats: No money, add/set custom money, ignore money, freebuilder mode, invent all, own all land. 
```

Comment:
A change log usually defines explicit changes that way people can track where a bug occurs if needbe. If I was to leave some out, someone may need to find the version where "freeholder mode" was implemented and would read all the change log but not find it.